src/cmd/prepare.go

```diff
@@ -30,7 +30,7 @@ var prepareTransformGitLinks = &cobra.Command{
 		host, fileName := args[0], args[1]
 
 		// Read the contents of the given file
-		content, err := ioutil.ReadFile(fileName)
+		content, err := os.ReadFile(fileName)
```

Comment:
Should we switch to os from now on? There are many other places ioutil still exists.

Reply:
Yeah the way we've dealt with other lint errors is to fix them when we touch a file vs go fix everywhere. I'd prefer to fix everywhere in another PR to keep this one less cluttered if that works.

Reply:
That works